packages/components/src/button/style.scss

```diff
@@ -246,7 +248,11 @@
 	&:disabled,
 	&[aria-disabled="true"] {
 		cursor: default;
-		opacity: 0.3;
+		color: $gray-600;
+
+		svg {
+			opacity: 0.66;
+		}
```

Comment:
This update from `0.3` to `0.6` opacity for SVG in disabled options should also probably be reviewed by the design team

Reply:
Any reason for the svg not to use the same color as the button text? That will eliminate any inconsistencies between text + icon when both are visible.

Reply:
I assume that this is intended to shift the SVG to have a similar rendering to the text color? This change isn't reflected in the screenshot shared in the PR; I think that this is too dark, and results in insufficient difference between active and disabled buttons. 

![image](https://github.com/user-attachments/assets/51a35d93-e687-42c7-85e6-c0d7d7a66576)

Image shows the undo/redo buttons; the redo button has the opacity of .66 applied, and the difference is barely noticeable.

Reply:
>  the redo button has the opacity of .66 applied, and the difference is barely noticeable.

@joedolson I'm not sure I'm seeing the same thing. Will comment on the PR